src/pybind/mgr/dashboard_v2/__init__.py

```diff
@@ -7,10 +7,28 @@
 import os
 
 if 'UNITTEST' not in os.environ:
+
+    class _LoggerProxy(object):
+        def __init__(self):
+            self.logger = None
+
+        def __getattr__(self, item):
+            if self.logger is None:
+                raise AttributeError("Logging not initialized")
+            return getattr(self.logger, item)
+
+
+    logger = _LoggerProxy()
+
     # pylint: disable=W0403,W0401
+
     from .module import *  # NOQA
 else:
+    import logging
     import sys
     # pylint: disable=W0403
     from . import ceph_module_mock
     sys.modules['ceph_module'] = ceph_module_mock
+    logging.basicConfig(level=logging.WARNING)
```

Comment:
Since this is for unit testing, shouldn't we actually set the level to DEBUG, so that we have the necessary info when something goes wrong?

Reply:
You haven't seen the INFO output of Cherrypy. :-/